parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -73,6 +73,32 @@ NVIDIA_DOCKER_VERSION=2.0.3
 DOCKER_VERSION=1.13.1-1
 NVIDIA_CONTAINER_RUNTIME_VERSION=2.0.0
 
+configure_prerequisites() {
+    if which apt > /dev/null; then
+        # Install required packages; only for apt-based systems
+        apt update -qq
+        apt install -y curl gpg jq nfs-common
```

Comment:
Let's move `curl`, `gpg`, and `nfs-common` into the `for apt_package` enumeration in the `installDeps` func (in `cse_install.sh`). `jq` is already there, so we should already get that on distros that don't pre-install it.

Reply:
Done!

Reply:
Correction, a few lines before we run `for apt_package` we run `curl`...
```
installDeps() {
    retrycmd_if_failure_no_stats 120 5 25 curl -fsSL https://packages.microsoft.com
...
    for apt_package in apache2-utils ... curl ...
```
So this will require some improvisation.

Reply:
I went ahead and added curl to the image so simplify things. However I uncovered a different issue. installDeps isn't actually getting called...

Reply:
OK, resolved the issue with installDeps - it's around detecting Debian derivatives as Debian while retaining Ubuntu-specific funtionality.